### ISSUE TITLE
Multiple Embeddings at the same time?

### ISSUE THREAD

**Issue by marketmind-schuster** [OkGoDoIt/OpenAI-API-dotnet#107]
Hi, 
first of all thanks for your amazing work!

What I'm trying to do is turning thousands of texts into embeddings. Currently that costs ~0.5 seconds per text if I transform each text separately. Is there any faster way of turning an array of strings into an array of embeddings vectors?

**Comment by edvanr** [comment]
Yeah looks like the OpenAI API takes in either a single string or an array of strings if you want to do multiple embeddings within one HTTP request
